docs/usage/self-hosted-configuration.md

```diff
@@ -292,6 +292,13 @@ Like this:
 
 ## dryRun
 
+You can choose from the following behaviors for the `dryRun` config option:
```

Comment:
Can you add a sentence above this one that explains what a "dry run" is, and why you might want to use it?

I guess it's a way to "preview" the behavior of Renovate, which doesn't actually make any changes to the repository files?